### PR TITLE
ci: add moe test

### PR DESCRIPTION
**Pull request by zhyncs**
Thank you for your contribution, we really appreciate it. The following instructions will help improve your pull request and make it easier to receive feedback. If there are any items you don't understand, don't worry. Just submit the pull request and ask the maintainers for help.

## Motivation

Please explain the motivation behind this PR and the goal you aim to achieve with it.

## Modification

Briefly describe the changes made in this PR.

## Checklist

1. Ensure pre-commit `pre-commit run --all-files` or other linting tools are used to fix potential lint issues.
2. Confirm that modifications are covered by complete unit tests. If not, please add more unit tests for correctness.
3. Modify documentation as needed, such as docstrings or example tutorials.

### CODE REVIEW

File: test/srt/test_moe_serving_throughput.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            res = run_benchmark(args)
+        finally:
+            kill_child_process(process.pid)
+
+        assert res["completed"] == num_prompts
+        return res
+
+    def test_default(self):
+        res = self.run_test(
+            disable_radix_cache=False,
```
Comment (merrymercy) [inline test/srt/test_moe_serving_throughput.py:63]: please sync with the new code https://github.com/sgl-project/sglang/blob/c877292cc12a61011694d7d0ea53c05f247003f6/test/srt/test_serving_throughput.py#L62-L93

File: test/srt/test_moe_serving_throughput.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            disable_flashinfer=False,
+            chunked_prefill_size=-1,
+        )
```
Comment (merrymercy) [inline test/srt/test_moe_serving_throughput.py]: sync with the new code style for the whole file
https://github.com/sgl-project/sglang/blob/c877292cc12a61011694d7d0ea53c05f247003f6/test/srt/test_serving_throughput.py#L62-L67

Comment (zhyncs) [reply test/srt/test_moe_serving_throughput.py]: ok

Comment (zhyncs) [reply test/srt/test_moe_serving_throughput.py]: Once the current MOE CI run is complete, I'll update it.

**Comment by zhyncs** [comment]
`mistralai/Mixtral-8x7B-Instruct-v0.1` tp 2 on A100 PCIE cc @merrymercy @Ying1123 @hnyls2002

File: .github/workflows/moe-test.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+name: MOE Test
```
Comment (merrymercy) [inline .github/workflows/moe-test.yml]:
```suggestion
name: MoE Test
```

**Review by merrymercy — APPROVED** [review]

File: .github/workflows/moe-test.yml (hunk 3 above)
Comment (zhyncs) [reply .github/workflows/moe-test.yml]: ok

**Review by hnyls2002 — APPROVED** [review]

**Pull request merged**